extensions/InputBox/InputBox.hooks.php

```diff
@@ -52,24 +52,38 @@ public static function onMediaWikiPerformAction(
 		$request, 
 		$wiki )
 	{
+		global $wgScript, $wgUser;
+
 		if( $wiki->getAction( $request ) !== 'edit' ){
 			# not our problem
 			return true;
 		}
+
+		$params = $request->getValues();
+
+		if ( $request->getText( 'preload', '' ) !== '' ){
```

Comment:
There should be a space between the closing parenthesis and the opening brace: `) {`.